src/core/postAttachments.ts

```diff
@@ -0,0 +1,52 @@
+import {tmpdir} from 'os';
+import {sep} from 'path';
+import joplin from 'api';
+import {Attachment} from '../model/attachment.model';
+import {AttachmentProperties} from '../model/attachmentProperties.mode';
+const fs = joplin.require('fs-extra');
+
+export class Attachments {
+    tempFolder: string = `${tmpdir}${sep}joplin-email-plugin${sep}`;
```

Comment:
consider using `path.join` instead

Reply:
Just noticed, you have a class `TempFolder` but still create the path manually here. Why not add a method to `TempFolder` instead? This way the path will only be defined in one place

Reply:
> consider using path.join instead

Done

> Just noticed, you have a class TempFolder but still create the path manually here. Why not add a method to TempFolder instead? This way the path will only be defined in one place

It's not for getting the temp path, it is for deleting previous attachment folders and creating new temp folders.
I renamed the class to 'setupTempFolder' to be more clear, and this class is called in the init of the plugin.

Reply:
The idea is you want to have the logic of getting the temp path only existing in one place. At the moment you have it duplicated here and in `setupTempFolder`.
Obviously in this case it's not a huge deal as this piece of code is very simple and is unlikely to change.